docs/how-to/index.md

```diff
@@ -16,3 +16,14 @@ yourself.
 
 install-chisel
 ```
+
+## Creating a chiselled Docker image
+
+Since Chisel can produce a minimal root file system, using it in Docker images
+has lots of value.
```

Comment:
same comment as in https://github.com/canonical/chisel-docs/pull/11/files#diff-282394d32e8fb2fc0286d001df1c7b5cdb1a3532b9a07d4516809dbb08cd1b79R3

Reply:
Fixed.